activitysim/examples/example_mtc/configs/free_parking.csv

```diff
@@ -7,3 +7,4 @@ util_income_high,High income housheold dummy,@(df.income>=60000) & (df.income<10
 util_hh_size_4_up,Household size is greater than 3 dummy,@df.hhsize>3,coef_hh_size_4_up,0.0
 util_more_autos_than_workers,More automobiles than workers dummy,@df.auto_ownership>df.num_workers,coef_more_autos_than_workers,0.0
 util_fewer_autos_than_workers,Fewer automobiles than workers dummy,@df.auto_ownership<df.num_workers,coef_fewer_autos_than_workers,0.0
+util_hh_has_ev,Household has an eletric vehicle,@df.has_ev,0.0,0.0
```

Comment:
typo `electric`

Reply:
Assume that it is ok that there isn't a coefficient here and it isn't doing anything in this model other than showing an example?

Reply:
yep, just a placeholder